.travis.yml

```diff
@@ -1,18 +1,16 @@
 language: elixir
 
 elixir:
-  - 1.5
-  - 1.6
-  - 1.7
+  - 1.14
```

Comment:
I don't think travis exists any more, can delete this file

Reply:
Good idea, I'll put that in a separate PR.

Any interest in a quick port to use the github [elixir action](https://github.com/actions/starter-workflows/blob/main/ci/elixir.yml) ?

Reply:
It is done :-)